common/compositor/vk/vkshim.h

```diff
@@ -16,6 +16,7 @@
 #define VKSHIM_H_
 
 #include <drm/drm_fourcc.h>
+#include <gbm.h>
```

Comment:
I've renamed GbmToVkFormat to NativeToVkFormat and moved to platfomrdefines.h. But I found DrmToVkFormat was not used at any place , I think GbmToVkFormat should be replaced with DrmToVkFormat at https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/drm/drmbuffer.cpp#L151,  right?